code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -28,12 +28,20 @@
 /obj/structure/closet/crate/can_close()
 	return TRUE
 
-/obj/structure/closet/crate/open()
+/obj/structure/closet/crate/open(irregular_open = FALSE)
 	if(src.opened)
 		return FALSE
 	if(!src.can_open())
 		return FALSE
 
+	if(!irregular_open)
```

Comment:
There are still quite some other cases list `bust_open` (not `burst_open`... yeah).
Also when one is EMPed. Or when the `effect/overload` processes. Or when the open door magic hits this.
Be sure to cover all the cases. Also cases where `usr` is null or such

Reply:
I inverted the check to be checking positive instead of negative.